imagersite/imagersite/views.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import unicode_literals
+from django.shortcuts import render, render_to_response
+from django.http import HttpResponse
+from django.template import Template
+from django.template import loader
+from django.views.generic import TemplateView
+from imager_images.models import Photo, Album
+from django.contrib.auth.models import User
+from django.conf import settings
+
+
+def home_view(request):
+    try:
+        pic_url = Photo.objects.filter(
+                  privacy='PU').order_by('?').first().file
```

Comment:
Also, you don't want to hand the file attribute back, you want the `url` of the file attribute:

```python
photo.file.url
```

That gives you a return value that is identical in type and usage to that for the case below.